testing/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vote.gov-cypress-testing-suite",
```

Comment:
went to update the name with what you suggested but it yelled at me for format and wanted it in this format